### PR TITLE
Put git metadata into unselectable spans

### PR DESCRIPTION
**Pull request by dbernheisel**
Currently, when trying to copy/paste the diffs from the browser into an editor, the line numbers and `-` and `+` are copied with it. I think generally folks don't want those when copying the diffs out of the browser.

This PR places that git metadata into unselectable spans. Since these adjust the HTML that is cached, I also bumped the cache_version which will bust the cache for all prior generated diffs. I didn't see a way to accomplish this without also changing the HTML.

image

Also comparing to GitHub:
image

I also implemented this same change at https://utils.zest.dev/gendiff if you'd like to try it there.

Here's the before capture for copy/paste
```
665676 | encode(msg, callback){
-- | --
666 | -     let payload = [
667 | -       msg.join_ref, msg.ref, msg.topic, msg.event, msg.payload
668 | -     ]
669 | -     return callback(JSON.stringify(payload))
677 | +     if(msg.payload.constructor === ArrayBuffer){
678 | +       return callback(this.binaryEncode(msg))
679 | +     } else {
680 | +       let payload = [msg.join_ref, msg.ref, msg.topic, msg.event, msg.payload]
681 | +       return callback(JSON.stringify(payload))
682 | +     }
670683 | },
```

Here's the after

```
  encode(msg, callback){
    let payload = [
      msg.join_ref, msg.ref, msg.topic, msg.event, msg.payload
    ]
    return callback(JSON.stringify(payload))
    if(msg.payload.constructor === ArrayBuffer){
      return callback(this.binaryEncode(msg))
    } else {
      let payload = [msg.join_ref, msg.ref, msg.topic, msg.event, msg.payload]
      return callback(JSON.stringify(payload))
    }
  },
```

And here's what GitHub copies as a comparison point:

```
  encode(msg, callback){
    let payload = [
      msg.join_ref, msg.ref, msg.topic, msg.event, msg.payload
    ]
    return callback(JSON.stringify(payload))
    if(msg.payload.constructor === ArrayBuffer){
      return callback(this.binaryEncode(msg))
    } else {
      let payload = [msg.join_ref, msg.ref, msg.topic, msg.event, msg.payload]
      return callback(JSON.stringify(payload))
    }
  },
```

### CODE REVIEW

**Comment by dbernheisel** [comment]
Unrelated to the PR, but if you may want to wait on busting the cache until this [git_diff issue is merged and released](https://github.com/mononym/git_diff/pull/8) so you don't have to bust the cache too often.

**Comment by ericmj** [comment]
I have rebased against `master` to get CI to run.

**Review by ericmj — APPROVED** [review]

**Review by supersimple — APPROVED** [review]
Thanks 🌯 🌮 🍻

**Pull request merged**
